rekall-core/rekall/plugins/filesystems/tsk.py

```diff
@@ -0,0 +1,159 @@
+import hashlib
+from time import sleep
+
+import pytsk3 as pytsk3
+import sys
+
+import plugin
+from rekall import obj
+from rekall.plugin import TypedProfileCommand, PhysicalASMixin, ProfileCommand
+from rekall.plugins import guess_profile
+
+
+class FSEntry(object):
+    def __init__(self, tsk_file):
+        self._tsk_file = tsk_file
```

Comment:
We should actually expose this (i.e. make it not private). It could be useful for retrieving more information about the file.